src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -255,7 +255,7 @@ def _get_vnet_network_profile(cmd, location, resource_group_name, vnet_name, vne
             subnet.delegations = [aci_delegation]
         else:
             for delegation in subnet.delegations:
-                if delegation.name != containerInstanceDelegationName:
+                if delegation.service_name != containerInstanceDelegationName:
```

Comment:
Don't see where `containerIntstanceDelegationName` is defined, but camel case isn't Pythonic. Should be snake case `container_instance_delegation_name`. Definitely agree with checking against `service_name` instead of `name`.